src/jit/emitxarch.h

```diff
@@ -333,6 +333,15 @@ void emitIns_R_R_A(instruction ins, emitAttr attr, regNumber reg1, regNumber reg
 
 void emitIns_R_R_AR(instruction ins, emitAttr attr, regNumber reg1, regNumber reg2, regNumber base, int offs);
 
+void emitIns_R_AR_R(instruction ins,
+                    emitAttr    attr,
+                    regNumber   reg1,
+                    regNumber   reg2,
+                    regNumber   base,
+                    regNumber   index,
+                    int         scale,
+                    int         offs);
```

Comment:
I don't know if this was formatted this way by jit-format, but I'd prefer to see it declared in a format more similar to those around it.

Reply:
Yes, this was given by jit-format, let me try to unify the format.

Reply:
This format is given by clang-format and cannot manually change...